### ISSUE TITLE
Multi-Store Support

### ISSUE THREAD

**Issue by hishamco** [simplcommerce/SimplCommerce#593]
This is will be a nice feature for the future if it's supported yet, the implementation need to be restructure little bit to use sort of multi tenancy .. again this may a great feature for 2.0.0

/cc @thiennn

**Comment by thiennn** [comment]
I am not sure whether it is a good idea as it will introduce a lot of complexity to the project. Majority users might not need this feature. 

For a real multi tenancy I prefer to handle it at the infrastructure level, I mean the container, each tenant is one or many container.

**Comment by HarleyBartles** [comment]
Yes to multi tenancy. Agreed that it needs a lot of thinking through. Pushing updates where something is overridden at tenant level could be problematic. Definitely would be a wonderful feature.

**Comment by rafsanulhasan** [comment]
Yes to multi tenancy. Though I agree with @thiennn that it would add more complexity, I think this an essential-to-have feature because in most of the cases, as far I dealt with customers, customers request for this feature because for a multi-region or multi-branch e-commerce system, multi tenancy is becoming increasingly requested feature nowadays...
